src/hub/atlas.js

```diff
@@ -421,6 +421,51 @@ class AtlasClient {
         return users;
     }
 
+    /**
+     * Update an existing users data
+     *
+     * @param {Object} json - A user data object as specified for the v1/user/ endpoint PATCH method
+     */
+    async updateUser(json) {
+        const op = { method: "PATCH", json};
+        await this.fetch("/v1/user/" + data.id, op);
```

Comment:
Is there anything useful in the response?  Might want to return whatever is in there if it's useful.